nbs/src/mfles.ipynb

```diff
@@ -0,0 +1,779 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.                results[i, j] = fastcos(x) / seasonal_period
</pre><p>Shouldnt' this be fastcos( x / seasonal_period)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/statsforecast/pull/810/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>